africanlii/templates/africanlii/layouts/document_detail.html

```diff
@@ -27,9 +27,19 @@ <h1 class="heading-underlined mb-4">{{ document.title }}</h1>
               <div class="col-md-6">
                 {% block document-metadata-content %}
                   <dl class="row">
-                    {% if document.authoring_body %}
+
+                    {% if document.jurisdiction %}
+                      <dt class="col-4">Jurisdiction </dt>
+                      <dd class="col-8 text-muted">{{ document.jurisdiction }} {% if document.locality %} · {{ document.locality }}</dd>{% endif %}
+                    {% endif %}
+
+                    {% if document.author %}
                       <dt class="col-4">Author</dt>
-                      <dd class="col-8 text-muted">{{ document.authoring_body }}</dd>
+                      <dd class="col-8 text-muted">{{ document.author }}</dd>
+                    {% endif %}
+
+                    {% if document.citation_links %}
+                      {{ citation_links|json_script:"citation-links" }}
                     {% endif %}
```

Comment:
This shouldn't be necessary. The old `citation_links` script on line 123 is sufficient.

Reply:
@longhotsummer I've reverted